client/src/main.ts

```diff
@@ -45,14 +45,16 @@ Vue.config.productionTip = true;
 
 djangoRest.restoreLogin(store).then(async () => {
   await store.dispatch.loadMe();
+  await store.dispatch.loadConfiguration();
   const user = store.state.me;
+  const { MIQAConfig } = store.state;
```

Comment:
inject/provide and the vuex are (loosely) two solutions to the same problem. Since it's available both from the store and via injection, there's some ambiguity as to where the "correct" place to get `MIQAConfig` from is. We could either just call `djangoRest` directly here and provide it, or not provide it and use `mapState` to get it from the store whenever it's needed.
(the same argument applies to `user`, that's probably a bigger refactor though)

This is just a gut feeling, I'm not that familiar with the pros/cons. Any thoughts @zachmullen ?

Reply:
My reasoning for this was the idea that we need to call `loadConfiguration` somewhere before render, just like we do for the user. It can be done in the `mounted` function, though, if we want.

Reply:
We discussed this in a team meeting and decided that keeping this information as an injected variable is an acceptable approach.